README.md

```diff
@@ -1,5 +1,95 @@
 # ActiveRecord CRUD Intro
 
+## Objectives
 
-1. Explain Create, Read, Update, Delete 
-2. Add codesnippets to the explanations as examples
\ No newline at end of file
+
+1. Understand CRUD - Create, Read, Update, and Delete.
+2. Get familiar with how to use ActiveRecord methods for Crud.
+
+## What is CRUD?
+
+CRUD is an acronym for the four verbs we use to operate on data: **C**reate, **R**ead, **U**pdate and **D**elete. Active Record automatically creates methods to allow an application to read and manipulate data stored within its tables.
+
+### Create
+
+To create new objects, we can either use the `new` or `create` method. 
+
+The `new` method will create a new object without any attributes.
+
+```ruby
+cat = Cat.new
+```
+To add attributes to our cat instance we need to write:
+
+```ruby
+cat = Cat.new
+cat.name = "Maru"
+cat.age = 3
+```
+The new method instantiates an object without saving it in the database. To persist the data we need to save it.
+
+```ruby
+cat = Cat.new
+cat.name = "Maru"
+cat.age = 3
+cat.save
+```
+
+The `create` Method will create an object with its attributes and save it to the database.
+
+```ruby
+cat = Cat.create(name: "Maru", age: 3)
+```
+
+### Read
+
+Active Record provides us with different methods to retrieve data from the database.
+
+To get a collection of all cats, we will use the `all` method.
+
+```ruby
+cats = Cat.all
+```
+
+If you only want to get the first entry in your database.
+
+```ruby
+cat = Cat.first
+```
+
+When you are looking for a specific attribute you can use the `find_by` method, but keep in mind this will only return the first entry that it finds.
+
+```ruby
+maru = Cat.find_by(name: "Maru")
+```
+
+Like SQL we can also use the `where` method in ActiveRecord.
+
+```ruby
+cat = Cat.where('age > 2')
+```
+
+### Update
+
+After retrieving an ActiveRecord object, we can also modify its attributes. 
+
+```ruby
+hannah = Cat.where(name: 'Hana') 
+hannah.name = "Hannah"
+hannah.save
+```
+Notice that after changing the attribute we also have to persist the new name in the database by using the save method. Or we could use the `update` method that comes with the ActiveRecord API.
+
+```ruby
+hannah = Cat.where(name: 'Hana') 
+hannah.update(name: "Hannah")
+```
+
+### Delete
+
+Likewise, after retrieving an ActiveRecord object, the object can be destroyed.
+
+```ruby
+hannah = Cat.where(name: 'Hannah') 
+hannah.destroy
+```
```

Comment:
Might be good to mention `destroy_all` here as well. 

"If we want to delete all the cats from out database, we can use the ActiveRecord's class method `.destory_all`. 

```ruby
Cat.destroy_all
```

This will delete each row in our table.